pmm-app/package.json

```diff
@@ -19,6 +19,7 @@
     "e2e": "docker run --net=host -v $PWD:/tests codeception/codeceptjs codeceptjs run-multiple parallel --all --debug --steps -c pr.codecept.js --grep '(?=.*)^(?!.*@visual-test)^(?!.*@not-pr-pipeline)'",
     "e2e:single": "codeceptjs run --steps  -c pr.codecept.js",
     "e2e:qan": "docker run --net=host -v $PWD:/tests codeception/codeceptjs codeceptjs run --steps  -c pr.codecept.js --grep '@new-qan' ",
+    "e2e:current": "docker run --net=host -v $PWD:/tests codeception/codeceptjs codeceptjs run --steps  -c pr.codecept.js --grep '@current' ",
```

Comment:
what are the tests in current group tag?

Reply:
I added this command for the convenient running of the test, I am currently working on, just by adding tag "current". I can remove it if you like.